src/droparea.js

```diff
@@ -203,7 +203,7 @@
               // check if dropped file is allowed
               if ( o.accepted !== null && typeof o.accepted != 'boolean' )
               {
-                if ( o.accepted.indexOf( file.name.substr( file.name.lastIndexOf('.') ) ) === -1 )
+                if ( o.accepted.indexOf( file.name.substr( file.name.lastIndexOf('.') ).toLowerCase() ) === -1 )
```

Comment:
Hi @tkrn ,

Thank you for your contribution. 👍 

What do you think to lower-case the `o.accepted` as well, when comparing to `name.lastIndexOf('.').toLowerCase()`? By doing that, we can ensure that accepted extensions are aways in lower-case too.

Can you add it to your code?

```
if ( o.accepted.toLowerCase().indexOf( file.name.substr( file.name.lastIndexOf('.') ).toLowerCase() ) === -1 )
```

Cheers,

Roger.